routes/slider.html

```diff
@@ -0,0 +1,119 @@
+      crossorigin="anonymous"
+    ></script>
+    <link rel="stylesheet" href="../index.css" />
+
+    <title>Subtle UI</title>
+  </head>
+  <body>
+    <div class="wrapper">
+      <nav class="navbar fixed">
+        <a href="../index.html" class="nav-heading">Subtle UI</a>
+        <ul class="nav-actions">
+          <li class="nav-action-item"><a href="../index.html">Home</a></li>
+          <li class="nav-action-item"><a href="../index.html">Docs</a></li>
+          <li class="nav-action-item">
+            <a href="https://github.com/bLAckBLAZER/Component-Library"
+              >Source</a
+            >
+          </li>
+        </ul>
+      </nav>
+      <main class="main-page">
+        <aside class="aside">
+          <ul class="aside-list">
+            <li class="list-item">
+              <a href="../index.html">Getting Started</a>
+            </li>
+            <li class="list-item"><a href="./avatar.html">Avatar</a></li>
+            <li class="list-item"><a href="./alert.html">Alert</a></li>
+            <li class="list-item"><a href="./badge.html">Badge</a></li>
+            <li class="list-item">
+              <a href="./buttons.html">Buttons</a>
+            </li>
+            <li class="list-item"><a href="./card.html">Card</a></li>
+            <li class="list-item"><a href="./image.html">Image</a></li>
+            <li class="list-item"><a href="./input.html">Input</a></li>
+            <li class="list-item"><a href="./navbar.html">NavBar</a></li>
+            <li class="list-item"><a href="./snackbar.html">Snackbar</a></li>
+            <li class="list-item">
+              <a href="./typography.html">Typography</a>
+            </li>
+            <li class="list-item">
+              <a href="./rating.html">Rating</a>
+            </li>
+            <li class="list-item">
+              <a href="./grids.html">Grids</a>
+            </li>
+            <li class="list-item">
+              <a href="./lists.html">Lists</a>
+            </li>
+            <li class="list-item">
+              <a href="./modal.html">Modal</a>
+            </li>
+            <li class="list-item active">
+              <a href="./slider.html">Slider</a>
+            </li>
+          </ul>
+        </aside>
+        <section class="content">
+          <div class="title">Slider</div>
+          <div class="heading">Introduction</div>
+          <p class="component-info">
+            You can also create a slider to take input which falls under some
+            range. Simply use class "slider" on the input tag of type "range"
+          </p>
+
+          <article class="article">
+            <div class="examples">
+              <input
+                type="range"
+                min="0"
+                max="100"
+                step="10"
+                value="33"
+                list="tickmarks"
+                class="slider"
+              />
+              <datalist id="tickmarks" class="slider-ticks">
```

Comment:
You can additionally add values which will help user to see where breakpoints are on your slider

Reply:
Thanks for the suggestion Hamza...added labels as well!